config/configtelemetry/configtelemetry.go

```diff
@@ -57,6 +58,11 @@ func (l Level) String() string {
 	return "unknown"
 }
 
+// MarshalText marshals Level to text.
+func (l Level) MarshalText() (text []byte, err error) {
+	return []byte(l.String()), nil
+}
```

Comment:
This one is needed because `UnmarshalText` expects a string value matching one of the `Level` strings. `Level` is an int32, so without the `MarshalText` function, the marshaler would marshal it as an int which would break the marshal/unmarshal cycle.